### ISSUE TITLE
Token Not Working with ACT & Is this bot still being managed??

### ISSUE THREAD

**Issue by yoloswaggyx** [castanhob/MognetPlugin#4]
When I put in the token into ACT and click apply the token disappear and everything is blank.  Also, if the developer is no longer managing this bot please update the description that it's no longer being updated, because I spent time downloading and installing the bot and token just to find out it doesn't work and now I found out your last post was months ago.

**Comment by castanhob** [comment]
Probably having the solution by today @yoloswaggyx

**Comment by castanhob** [comment]
@yoloswaggyx Probably solved after [0.2.0](https://github.com/castanhob/MognetPlugin/releases/tag/v0.2.0) release. Could you please confirm before I close the issue?

**Comment by luisk2094** [comment]
For me it works fine, the only thing is the labels on the configuration i think. I use them as follows to get this kind of notifications 
![imagen](https://user-images.githubusercontent.com/8525312/50407891-28ae7c00-07a6-11e9-8315-d4bce4b24343.png)
![imagen](https://user-images.githubusercontent.com/8525312/50407892-306e2080-07a6-11e9-9835-539251875d07.png)

**Comment by yoloswaggyx** [comment]
I will re-download and test.  Thanks

**Comment by yoloswaggyx** [comment]
I am able to connect with token and it shows my discord and channel.  Tested on striking dummy and it works.  Tested in 24man raid and there is an error.  As you can see, first boss it worked but anything after that was getting errors. 

![capture](https://user-images.githubusercontent.com/46036323/50419638-9ed2d300-07e6-11e9-9197-d1e61f361cdb.PNG)

**Comment by luisk2094** [comment]
@castanhob no, i meant like to get the max hit displayed i needed to check the critical hit label or something like that, still its not a big issue :D

**Comment by yoloswaggyx** [comment]
Thank you for the quick response.  I will keep testing for any other bugs I find.  I will also keep an eye out for the 24 man fix.  Thanks!

**Comment by yoloswaggyx** [comment]
@castanhob Hello, There seems to be an issue with the attribute selection and what is being posted by the bot.  We have tried removing the token and creating the token and it seems to not update with the new attribute when selected.  I posted screenshot of the discrepancy.  As you can see I selected direct hit% and crit% , but the bot post direct hit crit%. 

![capture](https://user-images.githubusercontent.com/46036323/50426490-b55b4780-0843-11e9-9937-7b2f485cdf2b.PNG)
![capture1](https://user-images.githubusercontent.com/46036323/50426492-b7bda180-0843-11e9-919a-1c5f3a484b34.PNG)

**Comment by luisk2094** [comment]
@yoloswaggyx yeah, that was what i meant. What i did to get the ones i wanted was testing one by one the labels until i got the desired values xD

**Comment by luisk2094** [comment]
yeah, now it works great. Also thanks for the hours filter, that seems useful
